lib/util/package-rules/repositories.ts

```diff
@@ -0,0 +1,19 @@
+import type { PackageRule, PackageRuleInputConfig } from '../../config/types';
+import { Matcher } from './base';
+import { anyMatchRegexOrMinimatch } from './match';
+
+export class RepositoriesMatcher extends Matcher {
+  override matches(
+    { repository }: PackageRuleInputConfig,
+    { matchRepositories }: PackageRule
+  ): boolean | null {
+    return anyMatchRegexOrMinimatch(matchRepositories, repository);
+  }
+
+  override excludes(
+    { repository }: PackageRuleInputConfig,
+    { excludeRepositories }: PackageRule
+  ): boolean | null {
+    return anyMatchRegexOrMinimatch(excludeRepositories, repository);
+  }
```

Comment:
Now I wonder do we really need to have these duplicated if they're just the opposite of each other?

Reply:
they can probably be simplified in a follow up PR